src/main/resources/library/attribute-definitions.json

```diff
@@ -4,14 +4,16 @@
   "title": "Library attribute definitions, v1",
   "description": "Constraints, facet definitions, and display definitions for FireCloud Library",
   "type": "object",
-  "required": ["library:datasetName", "library:datasetDescription", "library:datasetCustodian",
-               "library:datasetDepositor", "library:datasetOwner", "library:institute", "library:indication", "library:numSubjects", "library:projectName", "library:datatype",
+  "required": ["library:datasetName", "library:datasetDescription", "library:datasetCustodian", "library:datasetDepositor",
+               "library:datasetOwner", "library:institute", "library:numSubjects", "library:projectName", "library:datatype",
                "library:dataUseRestriction", "library:studyDesign", "library:cellType"],
-  "propertyOrder": ["library:datasetName", "library:datasetDescription", "library:datasetCustodian",
-                    "library:datasetDepositor", "library:datasetOwner", "library:institute", "library:indication", "library:numSubjects", "library:projectName", "library:datatype",
-                    "library:reference", "library:dataFileFormats", "library:technology", "library:profilingProtocol", "library:dataUseRestriction",
-                    "library:dataUseORSPConsentGroupNumber", "library:dataUseBSPSampleCollectionID", "library:studyDesign", "library:cellType", "library:coverage",
-                    "library:ethnicity", "library:primaryDiseaseSite", "library:broadInternalResearchProjectID", "library:broadInternalResearchProjectName",
+  "propertyOrder": ["library:numSubjects", "library:datatype", "library:dataUseRestriction", "library:diseaseOntology",
+                    "library:indication", "library:datasetName", "library:datasetDescription", "library:datasetCustodian",
+                    "library:datasetDepositor", "library:datasetOwner", "library:institute",  "library:projectName",
+                    "library:reference", "library:dataFileFormats", "library:technology", "library:profilingProtocol",
+                    "library:dataUseORSPConsentGroupNumber", "library:dataUseBSPSampleCollectionID", "library:studyDesign",
+                    "library:cellType", "library:coverage", "library:ethnicity", "library:primaryDiseaseSite",
+                    "library:broadInternalResearchProjectID", "library:broadInternalResearchProjectName",
```

Comment:
I was trying to get the order to work so that the diseaseOntology would be directly followed by indication in both edit and view mode, which Moran said she wanted. Then I realized that the order here doesn't really work because of the primary properties, which supersedes this -- only in the view mode. So I tried to change it so that the order would match in edit and view mode, because I thought that would make more sense.